docs/content/manual/release-specific/v1.7.0/test-rwx-fast-failover.md

```diff
@@ -0,0 +1,89 @@
+---
+title: RWX Fast Failover
+---
+
+## Related issues
+
+- https://github.com/longhorn/longhorn/issues/6205
+
+## LEP
+
+- https://github.com/longhorn/longhorn/pull/9069
+
+## Test step
+
+### Scale Test
+**Given** Longhorn cluster with 3 worker nodes.
+
+Make multiple deployments with a script such as
+```shell
+#!/bin/bash
+
+for i in {1..70}; do
+  cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: rwx-volume-pvc-$i
+  namespace: default
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Filesystem
+  storageClassName: longhorn-test
+  resources:
+    requests:
+      storage: 200Mi
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: app-$i
+  namespace: default
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: app-$i
+  template:
+    metadata:
+      labels:
+        app: app-$i
+    spec:
+      containers:
+      - name: app
+        image: busybox
+        command: ["/bin/sh", "-c", "exec 7<>/data/testfile-$(hostname -s); flock -x 7; while date | dd conv=fsync >&7 ; do sleep 1; done"]
+        volumeMounts:
+        - mountPath: /data
+          name: rwx-volume-pvc-$i
+      volumes:
+      - name: rwx-volume-pvc-$i
+        persistentVolumeClaim:
+          claimName: rwx-volume-pvc-$i
+EOF
+done
+```
+
+**Then** with the `rwx-enable-fast-failover` setting off, check the CPU and memory use totals for longhorn-manager and longhorn-share-manager pods, using something like `kubectl -n longhorn-system top pod`.  Establish baseline values.
+
+**Then** set `rwx-enable-fast-failover` to true, and scale down and up the deployments so that they will start updating and monitoring the leases.
+
+**Verify** that the CPU and memory use grows, but only by a small amount.
+
+**If possible** monitor the API server requests similar to the method in the report https://github.com/longhorn/longhorn/blob/master/scalability/reference-setup-performance-scalability-and-sizing-guidelines/public-cloud/medium-node-spec.md#longhorn-control-plane-performance
+
+**Verify** that the API request rate remains low.
+
+**Reference** How to set up a Grafana Testing Environment on Rancher
+https://github.com/longhorn/longhorn/issues/6205#issuecomment-2264430975
+
+### Failover Test
```

Comment:
Regarding  **Failover Test** this case, do we need to test both `hard` and `soft` mounts, or can we choose one based on our preference? Additionally, should we include the `auto-delete-pod-when-volume-detached-unexpectedly` setting, set to either false or true, in the combinations? Or is it okay to use the default value? Should this be included in the manual test case?

Based on my previous tests, I have tested all possible combinations as mentioned in 
 https://github.com/longhorn/longhorn/issues/6205#issuecomment-2257381697.

Reply:
Hopefully made that a little more clear in the latest iteration.